hack/run_ci_e2e_test.sh

```diff
@@ -34,7 +34,7 @@ if ! [ -x "$(command -v terraform)" ]; then
   fi
   echo "Installing terraform"
   cd /tmp
-  curl https://releases.hashicorp.com/terraform/0.11.10/terraform_0.11.10_linux_amd64.zip -Lo terraform.zip
+  curl https://releases.hashicorp.com/terraform/0.12.0/terraform_0.12.0_linux_amd64.zip -Lo terraform.zip
```

Comment:
```suggestion
  curl https://releases.hashicorp.com/terraform/0.12.2/terraform_0.12.2_linux_amd64.zip -Lo terraform.zip
```

Reply:
but probably I'd make that a variable, sourced from ENV, defaulted to `0.12.2` in case of absence.